addon/templates/components/common/bs-modal-simple.hbs

```diff
@@ -1,40 +1,41 @@
 {{#if inDom}}
   {{#maybe-in-element destinationElement _renderInPlace}}
-    {{#bs-modal/dialog
-      onClose=(action "close")
-      fade=fade
-      showModal=showModal
-      id=modalId
-      keyboard=keyboard
-      size=size
-      backdropClose=backdropClose
-      class=class
-      inDom=inDom
-      paddingLeft=paddingLeft
-      paddingRight=paddingRight
-      centered=(bs-eq position "center")
-    }}
-      {{bs-modal/header title=title closeButton=closeButton onClose=(action "close")}}
-      {{#bs-modal/body}}
+    <BsModal::dialog
+      @onClose={{action "close"}}
+      @fade={{this.fade}}
+      @showModal={{this.showModal}}
+      @keyboard={{this.keyboard}}
+      @size={{@size}}
+      @backdropClose={{this.backdropClose}}
+      @inDom={{this.inDom}}
+      @paddingLeft={{this.paddingLeft}}
+      @paddingRight={{this.paddingRight}}
+      @centered={{bs-eq this.position "center"}}
+      ...attributes
+      @class={{@class}}
```

Comment:
Any reason for the order of `...attributes` and `@class` here? I thought properties won't be affected anyways by `...attributes`. If I would prefer an order of having all properties before attributes for readability.

Reply:
Hm, no. I probably have used `class={{@class}}` before, and wanted to make sure that when setting `@class` this takes precedence of the splattributes. But as this is a prop again, this does not make really sense, will fix it!